[2024-10-18T17:57:24.934+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-18T17:57:24.960+0000] {__init__.py:51} DEBUG - Loading core task runner: StandardTaskRunner
[2024-10-18T17:57:24.979+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-18T17:57:24.988+0000] {base_task_runner.py:68} DEBUG - Planning to run as the  user
[2024-10-18T17:57:24.990+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [queued]> from DB
[2024-10-18T17:57:25.014+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2024-10-18T17:57:25.014+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-10-18T17:57:25.014+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2024-10-18T17:57:25.023+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-18T17:57:25.024+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [queued]>
[2024-10-18T17:57:25.024+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2024-10-18T17:57:25.024+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-10-18T17:57:25.029+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-18T17:57:25.035+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2024-10-18T17:57:25.035+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [queued]>
[2024-10-18T17:57:25.035+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-10-18T17:57:25.050+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): delete_btcusdt> on 2024-10-18 17:57:16.709448+00:00
[2024-10-18T17:57:25.055+0000] {standard_task_runner.py:57} INFO - Started process 19890 to run task
[2024-10-18T17:57:25.059+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'btcusdt_data_ingestion', 'delete_btcusdt', 'manual__2024-10-18T17:57:16.709448+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/ingest_btcusdt_data.py', '--cfg-path', '/tmp/tmpktcblthz']
[2024-10-18T17:57:25.062+0000] {standard_task_runner.py:85} INFO - Job 141: Subtask delete_btcusdt
[2024-10-18T17:57:25.063+0000] {cli_action_loggers.py:65} DEBUG - Calling callbacks: [<function default_action_log at 0x7f7ca9505550>]
[2024-10-18T17:57:25.125+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-18T17:57:25.137+0000] {task_command.py:410} INFO - Running <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [running]> on host 888d2c584607
[2024-10-18T17:57:25.138+0000] {settings.py:353} DEBUG - Disposing DB connection pool (PID 19890)
[2024-10-18T17:57:25.138+0000] {settings.py:212} DEBUG - Setting up DB connection pool (PID 19890)
[2024-10-18T17:57:25.139+0000] {settings.py:285} DEBUG - settings.prepare_engine_args(): Using NullPool
[2024-10-18T17:57:25.141+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [running]> from DB
[2024-10-18T17:57:25.189+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-18T17:57:25.198+0000] {taskinstance.py:868} DEBUG - Clearing XCom data
[2024-10-18T17:57:25.248+0000] {retries.py:80} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2024-10-18T17:57:25.265+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='btcusdt_data_ingestion' AIRFLOW_CTX_TASK_ID='delete_btcusdt' AIRFLOW_CTX_EXECUTION_DATE='2024-10-18T17:57:16.709448+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-18T17:57:16.709448+00:00'
[2024-10-18T17:57:25.266+0000] {__init__.py:117} DEBUG - Preparing lineage inlets and outlets
[2024-10-18T17:57:25.266+0000] {__init__.py:155} DEBUG - inlets: [], outlets: []
[2024-10-18T17:57:25.277+0000] {base.py:73} INFO - Using connection ID 'company_dw' for task execution.
[2024-10-18T17:57:25.294+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-10-18T17:57:25.296+0000] {__init__.py:75} DEBUG - Lineage called with inlets: [], outlets: []
[2024-10-18T17:57:25.296+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: btcusdt_data_ingestion.delete_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [running]> from DB
[2024-10-18T17:57:25.313+0000] {taskinstance.py:1356} DEBUG - Clearing next_method and next_kwargs.
[2024-10-18T17:57:25.313+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=btcusdt_data_ingestion, task_id=delete_btcusdt, execution_date=20241018T175716, start_date=20241018T175725, end_date=20241018T175725
[2024-10-18T17:57:25.313+0000] {taskinstance.py:2305} DEBUG - Task Duration set to 0.288959
[2024-10-18T17:57:25.327+0000] {cli_action_loggers.py:83} DEBUG - Calling callbacks: []
[2024-10-18T17:57:25.363+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-10-18T17:57:25.395+0000] {dagrun.py:697} DEBUG - number of tis tasks for <DagRun btcusdt_data_ingestion @ 2024-10-18 17:57:16.709448+00:00: manual__2024-10-18T17:57:16.709448+00:00, state:running, queued_at: 2024-10-18 17:57:16.779520+00:00. externally triggered: True>: 2 task(s)
[2024-10-18T17:57:25.397+0000] {dagrun.py:718} DEBUG - number of scheduleable tasks for <DagRun btcusdt_data_ingestion @ 2024-10-18 17:57:16.709448+00:00: manual__2024-10-18T17:57:16.709448+00:00, state:running, queued_at: 2024-10-18 17:57:16.779520+00:00. externally triggered: True>: 1 task(s)
[2024-10-18T17:57:25.398+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: btcusdt_data_ingestion.ingest_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-10-18T17:57:25.399+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: btcusdt_data_ingestion.ingest_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-18T17:57:25.401+0000] {taskinstance.py:1103} DEBUG - Dependencies all met for dep_context=None ti=<TaskInstance: btcusdt_data_ingestion.ingest_btcusdt manual__2024-10-18T17:57:16.709448+00:00 [None]>
[2024-10-18T17:57:25.405+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
